docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -914,29 +1189,29 @@ Destroy complete! Resources: 2 destroyed.
 <br />
 
 ####  Delete the Registry Server
-After deleting the cluster and cluster profile, navigate to **Tenant Settings** > **Registries** > **Pack Registries** to delete the registry service configuration from Palette.
-<br />
+
+After deleting the cluster and cluster profile, navigate to the **Tenant Settings** > **Registries** > **Pack Registries** section in Palette to remove the registry configuration.
 
 ![Screenshot of registry server delete in Palette](/tutorials/deploy-pack/registries-and-packs_deploy-pack_registry-delete.png)
 
-<br />
 
-Stop the registry server by closing the tutorials container bash session that serves the Ngrok reverse proxy server. At this point, you can close all the bash sessions. To remove the container and the image from the local machine, issue the following commands:
-<br />
+Now, delete the registry server. If you used the Spectro registry, stop the registry server by closing the tutorial container bash session that serves the Ngrok reverse proxy server. If you used the ECR registry, refer to the [Deleting a private repository](https://docs.aws.amazon.com/AmazonECR/latest/userguide/repository-delete.html) guide to delete the ECR repositories. <!--Finally, if you used a Basic registry, such as Harbor, delete your Harbor registry. If you hosted the Harbor registry on a virtual machine, for example, ensure to delete the virtual machine according to your setup.-->
```

Comment:
If the user created an AWS ECR registry with the AWS CLI through our guidance, then we should also help them delete the ECR registry.  Give them the commands to clean up the registry by removing the uploaded assets and then deleting the ecr registry. 

Remember, you have to remove everything from the ECR registry before AWS will allow you to delete it. Check out the [ecr-cleanup.sh](https://github.com/spectrocloud/docs-deploy-templates/blob/main/scripts/ecr-cleanup.sh) script I created in our template repo for an example of the commands. Keep in mind my script blows away EVERYTHING and no recommended for this use case where we need to specify the repository.

Reply:
I added the commands to delete the pack and the repositories, along with their output. However, the output is a little lengthy. What do you think?